generate_datafiles/compound_disease_datafiles.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+#Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmeddb"
+
+database_str = "postgresql+psycopg2://{}:{}@/{}?host=/var/run/postgresql".format(username, password, dbname)
+
+disease_url = 'https://raw.githubusercontent.com/dhimmel/disease-ontology/052ffcc960f5897a0575f5feff904ca84b7d2c1d/data/xrefs-prop-slim.tsv'
+compound_url = "https://raw.githubusercontent.com/dhimmel/drugbank/7b94454b14a2fa4bb9387cb3b4b9924619cfbd3e/data/drugbank.tsv"
+ctpd_url = "https://raw.githubusercontent.com/dhimmel/indications/11d535ba0884ee56c3cd5756fdfb4985f313bd80/catalog/indications.tsv"
+
+disease_ontology_df = pd.read_csv(disease_url, sep="\t")
+disease_ontology_df = (
+    disease_ontology_df
+    .drop_duplicates(["doid_code", "doid_name"])
+    .rename(columns={'doid_code': 'doid_id'})
+)
+
+drugbank_df = pd.read_table(compound_url).rename(index=str, columns={'name':'drug_name'})
+
+disease_ontology_df["dummy_key"] = 0
+drugbank_df["dummy_key"] = 0
+
+pair_df = (
+	disease_ontology_df
+	.merge(drugbank_df[["drugbank_id", "drug_name", "dummy_key"]], on='dummy_key')
+	.drop('dummy_key', axis=1)
+	)
+
+compound_treats_palliates_disease_df = (
```

Comment:
Whats the difference between these 3 DFs?

Reply:
So `compound_treats_palliates_disease_df` is a data frame that contains all indications of a compound affecting a disease either by treating it (DM) or by palliating it (SYM). Since both categories are combined, I gave this particular data frame the name.